### DISCUSSION TITLE
Social Preview

### DISCUSSION THREAD

**Discussion by D-Sketon** [hexojs/hexo#5455] [General]
The current hexo display looks like this
![image](https://github.com/hexojs/hexo/assets/49871906/13281bf8-d932-479e-90ad-8323abf70e58)
Maybe we could use the social preview feature like this to add a banner for hexo, which would be better publicized
![image](https://github.com/hexojs/hexo/assets/49871906/c75a5bbe-fcce-40bb-9005-4fa61c4375a6)

**Comment by uiolee** [comment]
Looks good

**Comment by D-Sketon** [comment]
I tried uploading one, and here's the current result. If anyone has better ideas, feel free to edit.
![image](https://github.com/hexojs/hexo/assets/49871906/c73d5d7c-4681-463f-ae11-2bf2a81f11b5)
![IMG_20240407_215413.jpg](https://github.com/hexojs/hexo/assets/49871906/20af0258-bd78-4b75-8629-2ae3e462520b)
